sync2jira/upstream_issue.py

```diff
@@ -309,26 +311,26 @@ def github_issues(upstream, config):
         issue['priority'] = ''
         orgname, reponame = upstream.rsplit('/', 1)
         issuenumber = issue['number']
-        default_github_project_fields = config['sync2jira']['default_github_project_fields']
-        project_github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
-        github_project_fields = default_github_project_fields | project_github_project_fields
+        github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
```

Comment:
We're now requiring _every_ project to provide the field mappings?  There's no common practice that we can put into a default?

Reply:
The logic has been changed in such a way that first check for the flag 'github_project_fields' inside 'issue_updates'. If it is present, the value for `github_project_fields` is expected.